src/main/java/com/adyen/service/checkout/RecurringApi.java

```diff
@@ -56,11 +56,10 @@ public RecurringApi(Client client, String baseURL) {
     * @param storedPaymentMethodId {@link String } The unique identifier of the token. (required)
     * @param shopperReference {@link String } Your reference to uniquely identify this shopper, for example user ID or account ID. Minimum length: 3 characters. &gt; Your reference must not include personally identifiable information (PII), for example name or email address. (required)
     * @param merchantAccount {@link String } Your merchant account. (required)
-    * @return {@link StoredPaymentMethodResource }
     * @throws ApiException if fails to make API call
     */
-    public StoredPaymentMethodResource deleteTokenForStoredPaymentDetails(String storedPaymentMethodId, String shopperReference, String merchantAccount) throws ApiException, IOException {
```

Comment:
`import com.adyen.model.checkout.StoredPaymentMethodResource;` on line 18 can also be removed?

Reply:
yes indeed